sdk/eventhub/event-hubs/src/batchingReceiver.ts

```diff
@@ -59,8 +59,13 @@ export class BatchingReceiver extends EventHubReceiver {
    * after creating the receiver object.
    * @ignore
    * @param maxMessageCount The maximum message count. Must be a value greater than 0.
-   * @param [maxWaitTimeInSeconds] The maximum wait time in seconds for which the Receiver
-   * should wait to receiver the said amount of messages. If not provided, it defaults to 60 seconds.
+   * @param [maxWaitTimeInSeconds] 
+   * 
+   * This will no longer be used.
```

Comment:
@ramya-rao-a Specifically anted to clarify if making this change to API is fine. This will subsequently require updates across source, tests and likely samples.

Reply:
Lets keep receiver changes out of scope of this PR and re-visit later